frontend/packages/kubevirt-plugin/integration-tests/tests/models/virtualMachine.ts

```diff
@@ -1,34 +1,58 @@
 /* eslint-disable no-await-in-loop, no-console */
+import { cloneDeep } from 'lodash';
 import { browser, ExpectedConditions as until } from 'protractor';
 import {
   waitForStringNotInElement,
   click,
   asyncForEach,
+  waitForStringInElement,
 } from '@console/shared/src/test-utils/utils';
 import { detailViewAction, listViewAction } from '@console/shared/src/test-utils/actions.view';
 import { VirtualMachineModel } from '@console/kubevirt-plugin/src/models';
+import { annotationDialogOverlay } from '@console/internal-integration-tests/views/modal-annotations.view';
 import * as vmView from '../../views/virtualMachine.view';
 import {
   VM_MIGRATION_TIMEOUT_SECS,
-  VM_ACTION,
-  TAB,
-  VM_STATUS,
   PAGE_LOAD_TIMEOUT_SECS,
-} from '../utils/consts';
+  VM_BOOTUP_TIMEOUT_SECS,
+} from '../utils/constants/common';
 import { BaseVirtualMachine } from './baseVirtualMachine';
 import { NodeSelectorDialog } from '../dialogs/nodeSelectorDialog';
 import { saveButton } from '../../views/kubevirtUIResource.view';
-import { annotationDialogOverlay } from '@console/internal-integration-tests/views/modal-annotations.view';
-import { MatchLabels } from '@console/internal/module/k8s';
+import { VMBuilderData } from '../types/vm';
+import { VM_ACTION, TAB, VM_STATUS } from '../utils/constants/vm';
+import { MatchLabels } from 'public/module/k8s';
+import { Wizard } from './wizard';
 
 const noConfirmDialogActions: VM_ACTION[] = [VM_ACTION.Start, VM_ACTION.Clone];
 
 export class VirtualMachine extends BaseVirtualMachine {
-  constructor(config) {
-    super({ ...config, model: VirtualMachineModel });
+  private data: VMBuilderData;
+
+  constructor(data: VMBuilderData) {
+    super({ ...data, model: VirtualMachineModel });
```

Comment:
can we just cherry pick the name and namespace here and not pass the whole object? It is easier to read that way IMO

Reply:
Yes